gpytorch/models/exact_prediction_strategies.py

```diff
@@ -84,6 +84,9 @@ def _exact_predictive_covar_inv_quad_form_root(self, precomputed_cache, test_tra
         """
         # Here the precomputed cache represents S,
         # where S S^T = (K_XX + sigma^2 I)^-1
+        if not torch.is_tensor(precomputed_cache):
+            # can't call tensor matmul on a lazy, so we'll flip things around
+            return precomputed_cache.transpose(-1, -2).matmul(test_train_covar.transpose(-1, -2)).transpose(-1, -2)
```

Comment:
@jacobrgardner, @gpleiss I wonder if it would make more sense to use a tensor and broadcast for the matmul, rather than using the BatchRepeatLazyTensor. Thoughts?